### PR TITLE
fixed files name

### PR DESCRIPTION
**Pull request by Tzach123**

### CODE REVIEW

File: src/pages/Home/Home.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
+import React, { useState, useEffect } from "react";
 
-import Header from "../../navigation/index.js";
-import ProductsListPage from "../ProductList/index.js";
-import ShoppingCartPage from "../ShoppingCart/index.js";
+import Header from "../../Navigation/Navigation.js";
+import ProductsListPage from "../ProductList/ProductList.js";
+import ShoppingCartPage from "../ShoppingCart/ShoppingCart.js";
 
 const Index = () => {
```
Comment (ariel22411) [inline src/pages/Home/Home.js:7]: Change the function name like a file name
From "Index" to "Home"

File: src/pages/ShoppingCart/ShoppingCart.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import react from 'react';
+import React from 'react';
 
 const index = () =>{
```
Comment (ariel22411) [inline src/pages/ShoppingCart/ShoppingCart.js:3]: Component function must start with a UpperCase  form index to Index,
and here you need to call this component "ShoppingCart"

**Review by ariel22411 — CHANGES_REQUESTED** [review]

File: src/pages/Home/Home.js (hunk 1 above)
Comment (Tzach123) [reply src/pages/Home/Home.js]: Done

File: src/pages/ShoppingCart/ShoppingCart.js (hunk 2 above)
Comment (Tzach123) [reply src/pages/ShoppingCart/ShoppingCart.js:3]: Done

File: src/App.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
-import react from 'react';
+import React from 'react';
 import './App.css';
 
-import Home from './pages/Home/index.js'
+import Home from './Pages/Home/Home.js';
 
 function App() {
```
Comment (ariel22411) [inline src/App.js:6]: Save the convention to all Components, 
if the all the components are arrow function change "function App" to "const App = () => {"

**Review by ariel22411 — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 3 above)
Comment (Tzach123) [reply src/App.js]: Done

**Pull request closed without merging**